nima_core/federation/hub_ner_worker.py

```
@@ -0,0 +1,291 @@
+
+        if not claimed:
+            return 0
+
+        processed_ids: set = set()
+        for row in claimed:
+            if self._stop_event.is_set():
+                break
+            hub_id, text, namespace, origin_node = row
+            try:
+                self._process_memory(int(hub_id), text, namespace, origin_node)
+            except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)